modules/claims_api/app/models/claims_api/veteran.rb

```diff
@@ -28,5 +29,31 @@ def va_profile=(new_va_profile)
       raise Common::Exceptions::ParameterMissing 'X-VA-Birth-Date' unless matches
       super(new_va_profile)
     end
+
+    def self.from_headers(headers, with_gender = false)
+      veteran = new(
+        ssn: ensure_header(headers, 'X-VA-SSN'),
+        loa: { current: :loa3 },
+        first_name: ensure_header(headers, 'X-VA-First-Name'),
+        last_name: ensure_header(headers, 'X-VA-Last-Name'),
+        va_profile: build_profile(headers),
+        edipi: ensure_header(headers, 'X-VA-EDIPI'),
+        last_signed_in: Time.zone.now
+      )
```

Comment:
Should `last_signed_in` be forced into UTC here?

Reply:
Done, changed to Time.now.utc